lib/main_jeos.pm

```diff
@@ -0,0 +1,200 @@
+# SUSE's openQA tests
+#
+# Copyright 2023 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: module loader for JeOS (Minimal-VM) tests
+# Maintainer: qa-c@suse.de
+
+package main_jeos;
+use Mojo::Base 'Exporter';
+use main_common;
+use main_containers qw(load_container_tests is_container_test);
+use version_utils qw(is_sle is_opensuse);
+use testapi;
+use utils;
+use Utils::Architectures;
+use Utils::Backends;
+
+our @EXPORT = qw(
+  load_jeos_tests
+);
+
+sub load_jeos_boot_tests {
+    my $machine = get_required_var('MACHINE');
+
+    loadtest 'installation/bootloader_svirt' if is_svirt;
+    loadtest 'installation/bootloader_uefi' if ($machine =~ /xen-hvm|hyperv$|vmware/);
```

Comment:
Better would be to use
https://github.com/os-autoinst/os-autoinst-distri-opensuse/blob/master/lib/Utils/Backends.pm